django_social_share/templatetags/social_share.py

```diff
@@ -111,3 +112,21 @@ def post_to_gplus(context, obj_or_url=None, link_text='Post to Google+'):
     context = post_to_gplus_url(context, obj_or_url)
     context['link_text'] = link_text
     return context
+
+
+@register.simple_tag(takes_context=True)
+def post_to_mail_url(context, text, subject, obj_or_url=None):
```

Comment:
Hi @fcurella, my rationale here was that a developer may want to leave the subject to user's discretion, since the user will be able to add it before sending the email. I don't mind changing the code as I personally always define a subject but I giving developers options is not bad either.